src/pages/recipes/AddIngredient.jsx

```diff
@@ -1,36 +1,15 @@
 import React from "react";
 import { useForm } from "react-hook-form";
-import * as yup from "yup";
 import { yupResolver } from "@hookform/resolvers/yup";
-import { setLocale } from "yup";
 import { useNavigate } from "react-router-dom";
 import propTypes from "prop-types";
 import { Input } from "../../components/Input/Input";
-import { Container } from "../../components/Container/Container";
 import { Button } from "../../components/Button/Button";
 import { Text } from "../../components/Text/Text";
 import { ButtonGroup } from "../../components/ButtonGroup/ButtonGroup";
+import { schema } from "../../utils/yup";
 
 function AddIngredient({ onSubmit }) {
-  setLocale({
-    mixed: {
-      required: "Pole ${label} jest wymagane.",
-    },
-    number: {
-      moreThan: "Pole ${label} musi mieć wartość większą od ${more}.",
-    },
-  });
-
-  const schema = yup.object({
```

Comment:
I think that the schema could stay here, closer to the form. `src/utils/yup.js` is a little bit to generic place and it may be hard to get to the point that the particular `schema` is related to the form from `AddIngredient` component. What do you think about that?